index/templates/index/base.html

```diff
@@ -48,7 +48,7 @@
     <link rel="stylesheet" href={% static "css/index.css" %} media="screen">
     <link rel="stylesheet" href={% static "css/mfp-fade.css" %} media="screen">
     <script src={% static "js/clock.js" %}></script>
-    <script src={% static "zeroclipboard/dist/ZeroClipboard.js"%}></script>
+    <script src={% static "zeroclipboard/dist/ZeroClipboard.min.js"%}></script>
```

Comment:
http://robertnyman.com/2008/04/23/where-to-include-javascript-files-in-a-document/